src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const oneDayPrice = 40;
+  const sevenDaysSale = 50;
+  const threeDaysSale = 20;
+
+  if (days >= 7) {
+    return days * oneDayPrice - sevenDaysSale;
+  }
+
+  if (days >= 3) {
+    return days * oneDayPrice - threeDaysSale;
+  }
```

Comment:
[checklist](https://github.com/mate-academy/js_task-transportation-on-vacation/blob/master/checklist.md) 1st item, I don't know what 7 and 3 are